spec/rest_in_peace/template_sanitizer_spec.rb

```diff
@@ -120,12 +120,20 @@
 
       context 'param which need to be encoded' do
         let(:params) { { id: 'this param need to be encoded' } }
-        let(:encoded_param) { 'this+param+need+to+be+encoded' }
+        let(:encoded_param) { 'this%20param%20need%20to%20be%20encoded' }
```

Comment:
I don't like that you changed the 'input' to one test.
If you would like to test, whether `'this%20param%20need%20to%20be%20encoded'` is parsed correctly as well, please create a specific second test for that case. But the original case should still work and should therefore still be tested..